src/ClassicUO.Client/LegionScripting/doc.md

```
@@ -2224,6 +2325,48 @@ This was generated on `6/11/2025`.
+
+ while True:
+ pb.SetProgress(cur, max)    
+ if cur >= max:
+ break
+ cur += 1
+ API.Pause(0.5)
+ ```
+
+
+**Parameters**  
+| Name | Type | Optional | Description |
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines
null

(MD058, blanks-around-tables)